hollow/src/main/java/com/netflix/hollow/api/common/VersionMinterWithCounter.java

```diff
@@ -14,9 +14,10 @@
  *     limitations under the License.
  *
  */
-package com.netflix.hollow.api.producer;
+package com.netflix.hollow.api.common;
```

Comment:
Hm do we think this will break people? I'm a little worried about the package rename causing runtime breakage

Reply:
I think this is worth calling out in release notes, but moving forward with anyway. No known usages that will break, and if any do exist they will be on the producer side.

Reply:
Backed this change out, since we're not parsing version string any more